src/lib/electron-renderer-tracker.ts

```diff
@@ -129,18 +133,55 @@ export function monitorRenderer(): void {
       return result;
     }
   };
+
+  ipcRenderer.sendSync = function (channel: string, ...args: any[]) {
+    const uuid = uuidv4(); // uuid is used to match the response with the request
```

Comment:
[randomUUID](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) is built into browsers and node which avoids the need for the extra module.
```suggestion
    const uuid = crypto.randomUUID(); // uuid is used to match the response with the request
```